target/linux/ath79/image/generic.mk

```diff
@@ -402,6 +402,27 @@ define Device/belkin_f9j1108-v2
 endef
 TARGET_DEVICES += belkin_f9j1108-v2
 
+define Device/belkin_f9k1115v2
```

Comment:
Please separate v2 with a hyphen here, like for the other device.

Reply:
just to double-check: you still prefer to have the dash (to make it consistent with other boards) even the device label (as state in the PR description) does not use a dash/space?

Reply:
I'm under the impression that the v2 represents a version/variant. In that case, it makes sense to separate it as we then could have a separate version field for metadata later without having to touch the name again. If that's not the case, and the v2 stands for something entirely different, one might reconsider.

Reply:
according to their download page this really is "version 2.0" hardware: https://www.belkin.com/us/support-article?articleNum=42287
it seems like v1 never made to the public.

I'll update the naming to be -v2 as suggested